v1/telemetry.js

```diff
@@ -1468,8 +1514,8 @@ Histogram.prototype.geometricStandardDeviation = function() {
     );
   }
   var count       = this.count();
-  var log_sum     = _aggregate(DataOffsets.LOG_SUM, this);
-  var log_sum_sq  = _aggregate(DataOffsets.LOG_SUM_SQ, this);
+  var log_sum     = this.precomputeAggregateQuantity(Telemetry.DataOffsets.LOG_SUM);
+  var log_sum_sq  = this.precomputeAggregateQuantity(Telemetry.DataOffsets.LOG_SUM_SQ);
```

Comment:
We use this to calculate the geometric means, as part of `Telemetry.Histogram.geometricMean()`.